### ISSUE TITLE
Bugged Icons

### ISSUE THREAD

**Issue by GsTio86** [DecentSoftware-eu/DecentHolograms#150]
### Just making sure

- [X] I made sure, this bug is not already known or reported.
- [X] I believe, that this bug is fixable.

### Reproduction

I thought it was Arclight causing the problem, but I actually tested CMI's holograms as well and they work fine without falling to the floor with re-reads
However, DecentHolograms can only be created normally for the first time, and the problem can be reproduced after reloading

Demo: https://gyazo.com/b795f2030d27fb435cd3bed411b9e7bc

### Server Version

1.16.5

### Client Version

1.16.5

### Plugin Version

2.8.3

### Log

https://paste.helpch.at/aluwexoyet.md

**Comment by Andre601** [comment]
Test it on a normal Paper Server and not Arclight. You claim it's not the issue, but the only proof you give is another plugin not failing, which is never a good way to figure out if a plugin is at fault here.

I'm confident that DH is not at fault here as every user reporting such an issue either used a Mod that changed item behaviour or used a server that simply is incompatible.
So please also test with a pure vanilla client if you're using client-side mods.

DH uses packets for all its stuff, including items, so these items do not exist on the server. perhaps this is the difference that causes CMI to not have the issue. Perhaps it uses actual, server-side items here.

tl;dr: Test it on a normal, supported server with a vanilla client. If it works there, then either the server or your client is at fault here.
